web/src/services/theme/reearthTheme/common/spacing.ts

```diff
@@ -1,3 +1,10 @@
+export const spacing = {
+  propertyMenuWidth: 308,
+  propertyMenuMinWidth: 200,
+  bottomPanelMinHeight: 136,
+  bottomPanelMaxHeight: 232,
+};
+
```

Comment:
I think these constants should be defined in `Editor` , should not be part of theme.